packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -85,12 +90,26 @@ if (!ExecutionEnvironment.canUseDOM) {
     clearTimeout(timeoutID);
   };
 } else {
-  // Always polyfill requestIdleCallback and cancelIdleCallback
+  let scheduledCallbackConfig: CallbackConfigType | null = null;
 
-  let scheduledRICCallback = null;
-  let isIdleScheduled = false;
-  let timeoutTime = -1;
+  // Number.MAX_SAFE_INTEGER is not supported in IE
+  const MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || 9007199254740991;
```

Comment:
<img width="463" alt="screen shot 2018-05-09 at 1 05 32 pm" src="https://user-images.githubusercontent.com/3624098/39836964-eb126ada-5389-11e8-97c1-d90f46ba00c1.png">

I don't think we need to account for the counter exceeding the maximum integer :D

Reply:
I never close tabs, man.

Reply:
Will remove, thanks!